website/docs/docs/dbt-cloud/on-premises/upgrading-kots.md

```diff
@@ -0,0 +1,103 @@
+---
+id: upgrading-kots
+title: Upgrading KOTS
+---
+
+## Overview
+
+As explained in the [dependencies](/docs/dbt-cloud/on-premises/dependencies) section, customer managed deployments rely on the Kubernetes-Off-The-Shelf (KOTS) appliance to configure and deploy the dbt Cloud application. When installing the dbt Cloud application for the first time as described in the [installation](/docs/dbt-cloud/on-premises/installation) section, the latest version of KOTS will be downloaded and installed along with the Admin Console. The KOTS version should be periodically upgraded to stay current with the [latest release](https://kots.io/release-notes/)
+
+To see the current installed version of KOTS the following command can be run:
+
+```bash
+kubectl kots version
+```
+
+The KOTS version is also visible at the bottom of the Admin Console UI next to "Terms" and "Privacy".
+
+<img src="/img/docs/dbt-cloud/deployment/kotsadm-version.png" />
+
+For newer versions of KOTS, the following command may be run to upgrade your installation to the latest stable version in place:
+
+```bash
+kubectl kots admin-console upgrade -n <namespace>
+```
+
+If you are upgrading your KOTS installation and have not tried the previous command, try this first before proceeding to the following section as manually upgrading may not be necessary. If the current installed version of KOTS is too old the upgrade command may fail and then the following instructions should be followed to manually upgrade the KOTS installation.
+
+## Manually upgrading the KOTS installation
+
+This section describes the steps needed to manually upgrade KOTS by deleting the current installation and re-installing the latest version. This process necesitates updating dbt Cloud to the latest stable version and as such is it is recommended that this process be executed along with a planned dbt Cloud upgrade. This process will not cause any downtime of the dbt Cloud application.
+
+Note: This process will erase the Version History in the Admin Console and force you to update to the latest stable version (meaning that you will lose the ability to roll back to prior versions). If you need to deploy a previous version of dbt Cloud please reach out to the Fishtown Analytics team.
```

Comment:
you should be more specific than `the Fishtown Analytics team` -- probably reach out to your account manager, or to support@getdbt.com